lib/agent0/agent0/hyperdrive/policies/arbitrage.py

```diff
@@ -48,24 +64,17 @@ def __init__(  # pylint: disable=too-many-arguments
             Random number generator
         slippage_tolerance: FixedPoint | None
             Slippage tolerance of trades
-        trade_amount: FixedPoint | None
-            The static amount to trade when opening a position
-        high_fixed_rate_thresh: FixedPoint | None
-            The upper threshold of the fixed rate to open a position
-        low_fixed_rate_thresh: FixedPoint | None
-            The lower threshold of the fixed rate to open a position
+        policy_config: Config | None
+            The custom arguments for this policy
         """
 
         # Defaults
-        if trade_amount is None:
-            trade_amount = FixedPoint(100)
-        if high_fixed_rate_thresh is None:
-            high_fixed_rate_thresh = FixedPoint("0.1")
-        if low_fixed_rate_thresh is None:
-            low_fixed_rate_thresh = FixedPoint("0.02")
-        self.trade_amount = trade_amount
-        self.high_fixed_rate_thresh = high_fixed_rate_thresh
-        self.low_fixed_rate_thresh = low_fixed_rate_thresh
+        # TODO the init function could probably just be the base class
```

Comment:
I think the setup you have is fine, I wouldn't bother with keeping this TODO around.